geoportailv3/templates/index.html

```diff
@@ -28,9 +28,46 @@
 
         <ul class="nav navbar-nav pull-right">
           <li class="visible-xs-inline-block search icon"><a href translate>search</a></li>
-          <li class="user icon"><a href translate>user</a></li>
+          <li class="user icon" ng-class="mainCtrl.userOpen ? 'active': ''">
+            <a href translate ngeo-btn ng-model="mainCtrl.userOpen" ng-click="mainCtrl.languageOpen = false">user</a>
+            <div ng-controller="UserController as userCtrl" class="navigation-account user-panel toolbox-panel toolbox-panel-top second-level">
```

Comment:
As discussed in the ngeo guidelines and during the workshop we want to limit the use of `ng-controller`. So, instead of using `ng-controller`, I would have created an `app-authentication` directive, with its own `AppAuthenticationController`, as done for other parts of the application.